sdk/keyvault-v2/azure-security-keyvault-secrets/customizations/pom.xml

```diff
@@ -8,15 +8,16 @@
     <description>This package contains code generation customization for Microsoft Azure Security Key Vault Secrets library</description>
 
     <groupId>com.azure.tools</groupId>
-    <artifactId>azure-security-keyvault-secrets-customization</artifactId>
+    <artifactId>azure-security-keyvault-secrets-codegen-customization</artifactId>
     <version>1.0.0-beta.1</version>
     <packaging>jar</packaging>
 
-    <dependencies>
+    <!-- TODO (vcolin7): Uncomment when this dependency is available in Maven Central -->
+    <!--<dependencies>
```

Comment:
Does this error if left uncommented?

Reply:
We can actually delete this for now. We'd need to add it if we ever needed to use these classes but on the TypeSpec package.